File: submission_server/sandbox/SUB_453/SRC.go
package main

import "fmt"

func fibb(n int64) int64 {
	if n < int64(2) {
		return int64(1)
	} else {
		n1, n2, curr := int64(1), int64(1), int64(2)
		for i := int64(2); i < n; i++ {
			n2 = n1
			n1 = curr
			curr = n1 + n2
		}
		return curr
	}
}

func main() {
	var n int64
	fmt.Scan(&n)
	for i := int64(0); i < n; i++ {
		var k int64
		fmt.Scan(&k)
		fmt.Println(fibb(k))
	}
}

File: submission_server/sandbox/SUB_455/SRC.cpp
#include <iostream>
#include <assert.h>
using namespace std;

long long fibb(long long n) {
	if (n < 2) {
		return 1;
	} else {
		long long n1 = 1;
		long long n2 = 1;
		long long i = 2;
		long long curr = n1 + n2;
		for (; i < n; i++) {
			n2 = n1;
			n1 = curr;
			curr = n1 + n2;
		}
		return curr;
	}
}

int main() {
	long long thang;
	cin >> thang;

	for (long long i = 0; i < thang; i++) {
		long long foo;
		cin >> foo;
		cout << fibb(foo) << endl;
	}

	return 0;
}